lib/rucio/daemons/hermes/hermes.py

```diff
@@ -337,7 +338,7 @@ def run_once_messages(heartbeat_handler: HeartbeatHandler,
     return must_sleep
 
 
-def stop(signum: int = None, frame: str = None) -> None:
```

Comment:
Here the type of `frame` is set to `str`. But according to this [discussion](https://github.com/python/typing/discussions/1042), it should be `FrameType`

Reply:
Which is why I found your choice of `*args` a nice compromise. The parameters aren’t used in the body of the function, so there is little to gain by being overly pedantic.